pyproject.toml

```diff
@@ -79,15 +82,14 @@ optional = true
 coverage = "^7.6.1"
 ops = "^2.17.1"
 pytest = "^8.3.4"
-pytest-lazy-fixture = "^0.6.3"
```

Comment:
I removed this one because the fixture functionality is included in `pytest` and installing it individually was causing the following error when I introduced parameterized tests:
```
 tests/unit/test_operator.py - AttributeError: 'CallSpec2' object has no attribute 'funcargs'
```